fast64_internal/oot/oot_actor.py

```diff
@@ -1,47 +1,410 @@
+		set=lambda self, value: setValues(self, value, 'YRot', 'actorID'))
+	propAnnotations['rotOverrideZ'] =  bpy.props.StringProperty(name = 'Rot Z', default = '0x0',
+		get=lambda self: getValues(self, self.actorID, None, 'ZRot', None),
+		set=lambda self, value: setValues(self, value, 'ZRot', 'actorID'))
+	propAnnotations['rotOverrideCustom'] = bpy.props.BoolProperty(name = 'Override Rotation', default = False)
+	propAnnotations['rotOverrideXCustom'] = bpy.props.StringProperty(name = 'Rot X', default = '0x0')
+	propAnnotations['rotOverrideYCustom'] = bpy.props.StringProperty(name = 'Rot Y', default = '0x0')
+	propAnnotations['rotOverrideZCustom'] = bpy.props.StringProperty(name = 'Rot Z', default = '0x0')
+
+	# We have to use a bool to know what's needed to be exported
+	propAnnotations['XRotBool'] = bpy.props.BoolProperty(default=False)
+	propAnnotations['YRotBool'] = bpy.props.BoolProperty(default=False)
+	propAnnotations['ZRotBool'] = bpy.props.BoolProperty(default=False)
+
+	# Transition Actors
+	propAnnotations['transActorID'] = bpy.props.EnumProperty(name='Transition Actor ID', items=ootEnumTransitionActorID)
+	propAnnotations['transActorIDCustom'] = bpy.props.StringProperty(name='Actor Key', default='ACTOR_CUSTOM')
+	propAnnotations['transActorKey'] = bpy.props.StringProperty(name='Transition Actor ID', default='0009')
+	propAnnotations['transActorParam'] = bpy.props.StringProperty(name = 'Actor Parameter', default = '0x0000', \
+		get=lambda self: getValues(self, self.transActorID, 'actorParam', 'Params', 'transActorParam'),
+		set=lambda self, value: setValues(self, value, 'Params', 'transActorID'))
+	propAnnotations['transActorParamCustom'] = bpy.props.StringProperty(name = 'Actor Parameter', default = '0x0000')
+
+	# Other
+	# isActorSynced is used to check if the blend's data is from an older version of Fast64
+	propAnnotations['isActorSynced'] = bpy.props.BoolProperty(default=False)
+	propAnnotations['itemChest'] = bpy.props.EnumProperty(name='Chest Content', items=ootChestContent)
+	propAnnotations['naviMsgID'] = bpy.props.EnumProperty(name='Navi Message ID', items=ootNaviMsgID)
+
+	# If you use the 'get=' option from Blender props don't actually save the data in the .blend
+	# When the get function is called we have to save the data that'll be returned
+	propAnnotations['paramToSave'] = bpy.props.StringProperty()
+	propAnnotations['transParamToSave'] = bpy.props.StringProperty()
+	propAnnotations['XRotToSave'] = bpy.props.StringProperty(default='0x0')
+	propAnnotations['YRotToSave'] = bpy.props.StringProperty(default='0x0')
+	propAnnotations['ZRotToSave'] = bpy.props.StringProperty(default='0x0')
+
+	# Collectible Drops List
+	itemDrops = [(elem.get('Value'), elem.get('Name'), \
+					elem.get('Name')) for listNode in root for elem in listNode if listNode.tag == 'List' \
+					and listNode.get('Name') == 'Collectibles']
+
+	# Generate the fields
+	for actorNode in root:
+		i = j = k = l = 1
```

Comment:
maybe i j k l should be `i_property`, `i_flag`, `i_bool`, `i_enum` since there's one for each of those. at first I though there was going to be a whole bunch of for loops monka